common/test/src/Model/TestGame.cpp

```diff
@@ -201,7 +206,7 @@ namespace TrenchBroom {
         }
 
         const Model::BrushFaceAttributes& TestGame::doDefaultFaceAttribs() const {
-            static const Model::BrushFaceAttributes defaults(Model::BrushFaceAttributes::NoTextureName);
+            static const Model::BrushFaceAttributes defaults(m_defaultFaceAttribs);
```

Comment:
(Accidentally made a comment using my work-related account... going to replace it here with a comment from my normal TB-related account to avoid confusion.)

This change to the test code means that the "defaults" returned by this function get locked into place the first time they are read. If you want to set the defaults to something then you have to call setDefaultFaceAttributes before they are used; any call to setDefaultFaceAttributes afterward will be ignored.

The more I think about this the more it bugs me a little. :-) I'm ok with leaving it this way, especially in test code, but if you want to suggest an alternative implementation I am all ears.

Reply:
Oh good point - this line should just return `m_defaultFaceAttribs`, right? No static construction etc.

Reply:
Yeah I had a moment of anxiety about whether one would be allowed to return a const reference in that case; these days I am not completely one with the zen of C++ const. I'm trying that now and it seems ok though.

Reply:
Yeah, returning the default face attribs by const ref is fine here.